app/portainer/views/stacks/edit/stackController.js

```diff
@@ -14,24 +14,28 @@ function ($q, $scope, $state, $transition$, StackService, NodeService, ServiceSe
     Endpoint: null
   };
 
+  $scope.duplicateStack = duplicateStack;
```

Comment:
Please keep the old controller function declaration here to avoid a mix of patterns.